### PR TITLE
New recipes

### PR DESCRIPTION
**Pull request by AndersonTorres**
New recipes:

- poet
- selectric-mode
- 0blayout
- 0xc
- @ 
- ass-mode
- olivetti
- recur
- tco
- oniisama (update)

### CODE REVIEW

File: recipes/0xc.rcp [hunk 1]
```diff
@@ -0,0 +1,4 @@
+(:name 0xc
+       :description "Base conversion made easy"
+       :type github
+       :pkgname "AdamNiederer/0xc")
```
Comment (stephan-cr) [inline recipes/0xc.rcp]: `:depends (s)` is missing.

File: recipes/ass-mode.rcp [hunk 2]
```diff
@@ -0,0 +1,4 @@
+(:name ass-mode
+       :description "Major mode for editing ASS/SSA subtitle files"
+       :type github
+       :pkgname "disbeliever/ass-mode")
```
Comment (stephan-cr) [inline recipes/ass-mode.rcp]: Depends on `s` as well.

File: recipes/tco.rcp [hunk 3]
```diff
@@ -0,0 +1,4 @@
+(:name tco
+       :description "Tail call optimisation in Emacs Lisp"
+       :type github
+       :pkgname "Wilfred/tco.el")
```
Comment (stephan-cr) [inline recipes/tco.rcp]: Depends on `dash`.

Comment (AndersonTorres) [reply recipes/tco.rcp]: Done!

Comment (stephan-cr) [reply recipes/tco.rcp]: Thanks a lot!

**Comment by yyr** [comment]
thanks.

**Pull request merged**